### DISCUSSION TITLE
Loading forever when manually starting livewire

### DISCUSSION THREAD

**Discussion by sinnbeck** [livewire/livewire#6284] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.0-beta.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Start livewire manually as described here https://livewire.laravel.com/docs/installation#manually-bundling-livewire-and-alpine
2. Add a simple div that shows loading when a request is made
3. Click a button with wire:click that does a request.

Now the loader will be running forever.

### Additional information

If I dont load my own js file, it works as expected.

I have filament installed, but it isn't active on the page I am testing with.

```html
<div>
    <div wire:loading.delay>Loading forever...</div>
    <button wire:click="doLoading">Foo</button>

</div>
```
```php
class Loadingtest extends Component
{
    public function doLoading()
    {

    }
}
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@sinnbeck you still need to add `@livewireStyles` to your layout. We've updated the docs 🙂

**Comment by Abhi0725** [comment]
Hi is this solved? I am facing the same issue when manually loading Livewire.

@sinnbeck @joshhanley

**Reply by sinnbeck** [reply]
@Abhi0725 Sadly I dont load it manually anymore. Decided to rewrite all logic to plain filament components.

If I later go back because I need some feature, I will let you know :)

**Reply by joshhanley** [reply]
@Abhi0725 you still need to include Livewire's styles in your head tag in your layout, if you're manually adding Livewire, so the loading styles work correctly.

Hope this helps!

**Reply by Abhi0725** [reply]
@joshhanley I have included the style as well but still seeing the loader forever.

![image](https://github.com/livewire/livewire/assets/8065382/fb7cceaf-a58a-4a5f-9e6f-1e9a35bb056c)

Also, even adding @livewireStyles doesn't help and my styles are ignored. I had to manually load via vite() as well.

**Reply by joshhanley** [reply, marked as answer]
@Abhi0725 Can you try putting livewire styles after the vite directive?

**Reply by Abhi0725** [reply]
that worked! thanks @joshhanley

**Reply by joshhanley** [reply]
@Abhi0725 Sweet! No worries 🙂 Yeah just the nature of how css works. You want Livewire's styles to have the highest priority, so needs to be loaded last.
